examples/example_1station_1event/README.md

```diff
@@ -0,0 +1,31 @@
+## Example 2: single station and event
+
+### Overview
+This example demonstrates the two central steps of this code:
+
+1. the computation of a database of Green functions for a specified region of the Earth.
+2. the computation of a kernel using the previously computed database
+
+### Instructions
+#### The Green function database
+To compute the database you need the two programs SGTpsv and SGTsh. That
+compute the databases necessary for PSV and SH waves, respectively. The sources
+of these codes are stored in `DSM-Kernel/src/SGTpsv-x.x-MPI` and `DSM-Kernel/src/SGTsh-x.x-MPI` where `x.x`
+stands for the version numbers.
+
+#### Compilation
+1. adjust the Makefiles (e.g. `DSM-Kernel/src/SGTpsv-x.x-MPI/Makefile`). You should for
```

Comment:
I added config.h so I will change also Makefile i.e. you do not have to touch Makefile any more